ntbs-service/Models/PaginationParameters.cs

```diff
@@ -7,5 +7,10 @@ public class PaginationParameters
     {
         public int PageSize;
         public int PageIndex;
+        public int? NtbsOffset;
+        public int? LegacyOffset;
+        public int? PreviousNtbsOffset;
+        public int? PreviousLegacyOffset;
+        public int NumberOfNtbsNotificationsToFetch => NtbsOffset != null ? PageSize : PageIndex * PageSize;

```

Comment:
Using this so I can still use PaginationParameters in GetPaginatedItems in SearchService, could move the logic to that function